doc/services/storage/zms/zms.rst

```diff
@@ -414,6 +415,41 @@ verified to make sure that the application could work with one subsystem or the
 both solutions could be implemented, the best choice should be based on the calculations of the
 life expectancy of the device described in this section: :ref:`wear-leveling`.
 
+Recommendations to increase performance
+***************************************
+
+Sector size and count
+=====================
+
+- The total size of the storage partition should be well dimensioned to achieve the best
+  performance for ZMS.
+  All the information regarding the effectively available free space in ZMS can be found
+  in the documentation. See :ref:`free-space`.
+  We recommend choosing a storage partition that can hold double the size of the key-value pairs
+  that will be written in the storage.
```

Comment:
```suggestion
  It's recommended to choose a storage partition size that is double the size of the key-value pairs
  that will be written in the storage.
```
Do we have to count the key sizes (32 bits) in that? It's a bit unclear.

Reply:
Actually we have to count the total size of the key-value pair. whether it is a small data value (16 bytes for each key/value pair) or large data value (16 bytes + sizeof(data))